tests/integration/common.sh

```diff
@@ -90,17 +90,16 @@ start_otbr() {
     pidof otbr-agent && die "killing otbr-agent failed"
 
     local ncp=$1
-    local backbone_if=$2
 
     sudo chmod 777 ${WPANTUND_CONF}
     sudo echo "Config:NCP:SocketPath \"system:${ncp} 1\"" > ${WPANTUND_CONF}
 
-    sudo wpantund -c ${WPANTUND_CONF} -d 7 > ${WPANTUND_LOG} 2>&1 &
+    sudo wpantund -c ${WPANTUND_CONF} -d 7 > "${WPANTUND_LOG}" 2>&1 &
```

Comment:
Should also quote `$WPANTUND_CONF` ?

Reply:
I believe it is because shellcheck knows that there is no word splitting iwith `$WPANTUND_CONF` (`$WPANTUND_CONF` has a constant initializer).